### ISSUE TITLE
[Tensorboard] `RuntimeWarning: invalid value encountered in multiply` when calling `add_embedding` with `label_img`.

### ISSUE THREAD

**Issue by amoshyc** [pytorch/pytorch#34954]
## 🐛 Bug

When calling [`add_embedding`](https://pytorch.org/docs/stable/tensorboard.html#torch.utils.tensorboard.writer.SummaryWriter.add_embedding) with `label_img`, sometimes I get following warning:
```
.../python3.7/site-packages/torch/utils/tensorboard/_embedding.py:37: RuntimeWarning: invalid value encountered in multiply
  im = Image.fromarray(np.uint8((arranged_augment_square_HWC * 255).clip(0, 255)))
```

I suspect it is caused by [L34](https://github.com/pytorch/pytorch/blob/73c1030328deb6f00ab867f5c402b203dbffae3c/torch/utils/tensorboard/_embedding.py#L34) in `make_sprite`.

1. According to the [doc](https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.html), the result of `np.ndarray(shape=(...))` may contain NaN.

2. Only part of the elements of `arranged_augment_square_HWC` will be replaced with `arranged_img_HWC`. Thus, some NaN may remain in `arranged_augment_square_HWC`.

3. Therefore [L37](https://github.com/pytorch/pytorch/blob/73c1030328deb6f00ab867f5c402b203dbffae3c/torch/utils/tensorboard/_embedding.py#L37) trigger the RuntimeWarning.

## To Reproduce

Although I encounter this warning multiple times, I'm not able to reproduce it in a small example, sorry. It seems that the value of `np.ndarray()` is random and not depends on `np.random.seed()`. 

## Expected behavior

No warning is raised.

## Environment

```
Collecting environment information...PyTorch version: 1.4.0
Is debug build: NoCUDA used to build PyTorch: 10.0
OS: Ubuntu 16.04.6 LTS
GCC version: (Ubuntu 7.4.0-1ubuntu1~16.04~ppa1) 7.4.0
CMake version: version 3.5.1

Python version: 3.7
Is CUDA available: Yes
CUDA runtime version: 10.0.130
GPU models and configuration: 
GPU 0: TITAN X (Pascal)
GPU 1: TITAN X (Pascal)

Nvidia driver version: 410.129
cuDNN version: /usr/local/cuda-9.1/targets/x86_64-linux/lib/libcudnn.so.7

Versions of relevant libraries:
[pip] numpy==1.18.1
[pip] torch==1.4.0
[pip] torchvision==0.5.0
[conda] blas                      1.0                         mkl  
[conda] mkl                       2019.4                      243  
[conda] mkl-service               2.3.0            py37he904b0f_0  
[conda] mkl_fft                   1.0.15           py37ha843d7b_0  
[conda] mkl_random                1.1.0            py37hd6b4f25_0  
[conda] pytorch                   1.4.0           py3.7_cuda10.0.130_cudnn7.6.3_0    pytorch
[conda] torchvision               0.5.0                py37_cu100    pytorch
```

## Additional context

I had inserted

```python
print(np.isnan(arranged_augment_square_HWC).sum())
```
before [L37](https://github.com/pytorch/pytorch/blob/73c1030328deb6f00ab867f5c402b203dbffae3c/torch/utils/tensorboard/_embedding.py#L37) and did observe non-zero value sometimes.

And changing `np.ndarray()` to `np.zeros()` seems to resolve the warning.

**Comment by colesbury** [comment]
Thanks for tracking this down. A PR fixing this would be very welcome.

**Comment by lanpa** [comment]
Thanks @amoshyc
